shoop_setup_utils/commands.py

```diff
@@ -82,9 +86,29 @@ class BuildProductionResourcesCommand(BuildResourcesCommand):
     clean = True
 
 
+class BuildTranslationsCommand(distutils.core.Command):
+    command_name = 'build_translations'
+    description = 'compile message catalogs to MO files via Django compilemessages'
+    user_options = []
+
+    def initialize_options(self):
+        pass
+
+    def finalize_options(self):
+        pass
+
+    def run(self):
+        os.chdir("shoop")
+        settings.configure()
```

Comment:
This just reinforces my opinion that the translation thing should definitely not be a management command.